recipes/libsolace/all/conandata.yml

```diff
@@ -5,4 +5,10 @@ sources:
   "0.3.7":
     sha256: 7f9790f850a0d7209f95be03e1c58e67f91faeec2e12709893cf32e5f09aa6d0
     url: https://github.com/abbyssoul/libsolace/archive/v0.3.7.zip
+  "0.3.8":
```

Comment:
Do you really need the `.8` version or the latest patch `.9` would work?

Reply:
I realized that I forgot to publish .8
So yeah - .9 should be enough but I thought I should include .8 to fix the omission.

Reply:
Don't worry. We try not to package all the patches of the libraries unless it is required by a dependency or specifically requested by the community. thanks!

Reply:
@abbyssoul please remove the .8 version if not needed. I will merge the PR then. Thanks!

Reply:
done.